Unify Formatting (#82)
C2Devel/terraform-provider-croccloud: website/docs/d/ebs_volumes.md

layout: "aws"
page_title: "aws_ebs_volumes"
description: |-
    Provides identifying information about EBS volumes matching given criteria.
---

# Data Source: aws_ebs_volumes

> Provides identifying information about EBS volumes matching given criteria.

Reviewer: ```suggestion
Provides a list of EBS volume IDs matching given criteria.
```

Нужно продублировать в служебный блок

Author: заменил артикль на the, где есть уточнение, что за список

layout: "aws"
page_title: "aws_ebs_volumes"
description: |-
    Provides a list of EBS volume IDs.
---

[describe-volumes]: https://docs.k2.cloud/en/api/ec2/volumes/DescribeVolumes.html

> [tf-ebs-volume]: ebs_volume.html

Reviewer: Можно удалить

}
```

## Argument Reference

* `filter` - (Optional) One or more name/value pairs to use as filters.

>     * _Valid values:_ See supported names and values in [EC2 API documentation][describe-volumes]

Reviewer: Нужно перенести определение ссылки в начало страницы

}
```

## Argument Reference

* `filter` - (Optional) One or more name/value pairs to use as filters.
    * _Valid values:_ See supported names and values in [EC2 API documentation][describe-volumes]
* `tags` - (Optional) Map of tags, each pair of which must exactly match
  a pair on the desired volumes.

> ## Attribute Reference

Reviewer: ```suggestion
## Attribute Reference

In addition to all arguments above, the following attributes are exported:
```
